src/app/stacktrace.h

```diff
@@ -9,16 +9,18 @@
 #include <execinfo.h>
 #include <cxxabi.h>
 
+#include <vector>
+
 /** Print a demangled stack backtrace of the caller function to FILE* out. */
 static inline void print_stacktrace(FILE *out = stderr, unsigned int max_frames = 63)
 {
     fprintf(out, "stack trace:\n");
 
     // storage array for stack trace address data
-    void *addrlist[max_frames + 1];
```

Comment:
Yes, C++ does not support variable length array and with `-pedantic` GCC rejects this.